packages/cli/src/commands.js

```diff
@@ -140,7 +148,13 @@ export async function importCommands() {
       pkgs.set(pkg.name, () => Promise.all(
         pkg['@percy/cli'].commands.map(async cmdPath => {
           let modulePath = path.join(pkgPath, cmdPath);
-          let module = await import(url.pathToFileURL(modulePath).href);
+          // Below code is used in scripts/build.sh to update href
+          let module = null;
+          if (process.env.ENVIRONMENT === 'executable') {
```

Comment:
hmm, this would mean we'll have to package .env as well in executable.

what we want is while building the code, default skip/remove the codepath for else {}

lets think more about the solutioning here.

Reply:
yes we will have to create a .env file and use dotenv package. that should work

Reply:
yeah, we already have the package under `percy/env` we can extend it there (`packages/env/src/environment.js`)

Reply:
.env will work only when file is present from location we are executing, i have created a new file with will be entrypoint for executable and will set nodeenv